examples/tf/ppo_pendulum.py

```diff
@@ -71,6 +71,7 @@ def tf_ppo_pendulum(ctxt=None, seed=1):
             entropy_method='max',
             policy_ent_coeff=0.02,
             center_adv=False,
+            fit_baseline_order='after',
```

Comment:
please call this `fit_baseline`

Reply:
there's already a function called `_fit_baseline`

Reply:
then rename it. the public API is more important.